modules/theme/src/themes/wso2is/assets/images/illustrations/ai-loading-screen-placeholder.svg

```diff
@@ -0,0 +1,111 @@
+<svg width="458" height="464" viewBox="0 0 407 412" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Let's add the license header here as well.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/5961/commits/891f646b2c63094d4467060253aa82c4bd6751b5